test/schema.rb

```diff
@@ -6,20 +6,23 @@
   end
 
   create_table :access_control_permitted_actions, force: true, id: :uuid do |t|
-    t.column :segment_id, :integer
+    t.column :segment_id, :integer, default: -1
```

Comment:
The table indexes need a sensible default on segment_id for the unique table index